server/src/utils/slugify.js

```diff
@@ -0,0 +1,4 @@
+const punycode = require('punycode')
```

Comment:
Seems like the core `punycode`-module is being deprecated: https://nodejs.org/api/punycode.html . Should probably add it as a dependency and import with `punycode/` https://github.com/bestiejs/punycode.js?PHPSESSID=5790fa7ffb5d909fb63c31c0f6571035

Reply:
Good catch!